targets/TARGET_STM/mbed_crc_api.c

```diff
@@ -14,6 +14,7 @@ bool hal_crc_is_supported(const crc_mbed_config_t *config)
         return false;
     }
 
+#if !defined(STM32L4)
```

Comment:
Maybe we can keep the test of the 4 config-> width for the L4 family (32, 16, 8 and 7) ?